### PR TITLE
Added Veritree Donation 24.10.2021

### PR DESCRIPTION
**Pull request by ResponsibleStaking**

### CODE REVIEW

File: VITAL/donations.yml [hunk 1]
```diff
@@ -32,3 +32,18 @@
...
+  recipient: &veritree
+    name: Veritree Cardano Forest
+    address: 230-1275 Venables Street Vancouver, BC V6A 2E4
+    tel: 1-833-520-1145
+    contact: info@veritree.com
+  ada: 500
+  usd: 1060
+  timestamp: 1635036960
+  files:
+    - VITAL/lobby16/2021-06-25/PaymentConfirmation.pdf
```
Comment (nova4005) [inline VITAL/donations.yml:46]: This is the same file from the previous donation. If you could update this with the correct file the rest looks good.

You may also need to update your repo with the main branch to make sure it is not out of date. Instructions are in the [README.md](https://github.com/CardanoMDP/Donations#keeping-your-repository-up-to-date-with-the-upstream-repository-cardanomdpdonations)

Comment (ResponsibleStaking) [reply VITAL/donations.yml]: Removed the file as Veritree is purely documented in the Cardano Transaction.
Thanks for working on the PRs!

**Comment by ResponsibleStaking** [comment]
HI @nova4005 !
I updated the PR with the missing file and another Donation ;)
Regards,
Markus

**Review by nova4005 — APPROVED** [review]
LGTM

**Pull request merged**